docs/application/vscode-ext/web.md

```diff
@@ -0,0 +1,306 @@
+>
+> In addition, vscode terminal displays successful launch log messages:
+> 
+> 1. debug 0 means normal launch.
+> 2. debug 1 means launching an Inspector available.
+
+### Debug Application in Emulator
+
+To debug application, follow these steps:
+
+> **Note**
+>
+> Ensure the following:
+> - Emulator is launched
+> - Device is connected to the PC
+> - Chrome browser is installed
+
+1. Open **Command Palette** and enter **Debug Tizen App**.
+2. Enter the name of the Tizen Web App you want to run.
+3. A successful popup is created and the app is launched.
+
+> **Note**
+> 
+> In addition, `vscode` terminal displays successful launch log messages:
+> vscode Terminal displays a successful launch log
+>
+> - debug `0` means normal launch.
+> - debug `1` means launching an inspector available from command line.
+> - Additionally, debug the connected port as shown in the following log.
+
+  ![Debugging you application](media/tizen_web16.png)
+
+4. Connect **Debug Tizen App** with **Inspector**.
+5. Open **Command Palette** and enter **Run Inspector**.
+6. Enter the port number on your local PC, for example, 9999
+7. In Chrome browser, the Inspector automatically launches and associates with the launched app.
+
+    ![Debugging you application](media/tizen_web17.png)
+
+### Debug Application in Web Simulator
+
+To debug application, follow these steps:
+
+1. Open **Command Palette** and enter **Run Web Simulator**.
+2. After the Web Simulator is launched, enter the path of the file in the box. For example, file:///C:/workspace/basic/index.html.
+
+    ![Debugging you application in the Web Simulator](media/tizen_web18.png)
+
+## Tizen Web with Code IntelliSense
+
+VS Code Extension for Tizen Web supports intelliSense, which includes the following features:
+
+- Code Completion
+- Hover
+- Signature Help
+- Document View document
+
+You can use these features to easily develop application using TAU.
+
+### Features
+
+- Code completion
+  
+  Display a list of recommended values when specifying attributes or APIs.
+
+    ![Code completion](media/tizen_web20.gif)
+
+- Hover
+  
+  Display a description of the attribute or API when the mouse over attritubes
+
+    ![Hover](media/tizen_web21.gif)
+
+- Signature Help
+  
+  Displays an information of API parameters when using the API
+
+    ![Signature Help](media/tizen_web22.gif)
+
+- Document Viewer
+  
+  Displays a document related with attributes or APIs.
+    > **How to use the Document Viewer**
+    > 
+    > Click the "Search in TAU document" on the context menu
+
+    ![Document Viewer](media/tizen_web23.gif)
+
+
+## Troubleshooting
+
+This section contains instructions for common problems with the Tizen Baseline Web IDE.
+
+- To troubleshoot red screen error, follow these steps:
+
+  a. Launch **Web Simulator**. 
+  b. When the red screen is created, press F12.  
+  c. Click Application Tab at the top-end immediately after the Chrome Devtool opens. 
```

Comment:
@mijin85cho,

Please can you verify the suggested rephrase:
**Application** tab at the top after the Chrome Devtool opens. 
or
Click the **Application** tab at the top of the window after the Chrome Devtool opens.

Since, I cannot see the UI, so can you help me verifying the above sentence?

Reply:
@sabeena21 
"**Application** tab at the top after the Chrome Devtool opens." would work.

I think you can see the UI also if you press **F12** key on Chrome window.

Reply:
@mijin85cho, okay
@DohyungLim, Please can you change to:
**Application** tab at the top after the Chrome Devtool opens.

Reply:
done.